ts/src/gate.ts

```diff
@@ -3916,6 +3918,7 @@ export default class gate extends Exchange {
             'margin': 'privateSpotGet' + methodMiddle + 'OrderId',
             'swap': 'privateFuturesGetSettle' + methodMiddle + 'OrderId',
             'future': 'privateDeliveryGetSettle' + methodMiddle + 'OrderId',
+            'option': 'privateOptionsGetOrdersOrderId',
```

Comment:
@Dan-krm we're missing option here 
![image](https://github.com/ccxt/ccxt/assets/43336371/5ed423f7-3e33-4838-9d40-bbc08000484b)

Reply:
Also, when stumbling on a method without js docs, let's add it pls

Reply:
hum, does it only return open orders? 
```
 n gate fetchOrder "2597550537" "BTC/USDT:USDT-230601-27500-C" --verbose
2023-05-31T10:37:15.878Z
Node.js: v18.14.0
CCXT v3.1.15
gate.fetchOrder (2597550537, BTC/USDT:USDT-230601-27500-C)
fetch Request:
 gate GET https://api.gateio.ws/api/v4/options/orders/2597550537?contract=BTC_USDT-20230601-27500-C 
RequestHeaders:
 {
  'X-Gate-Channel-Id': 'ccxt',
  KEY: 'd7fc5d5c2bdda0964035ec4bb1d8ddaa',
  Timestamp: '1685529439',
  SIGN: 'ae392732ddc23e06bba3e9f39875e11f01435bb68fadc4fd776f98f705f61c5d1b46866d68821aeca096cfe2167071fa0b0bacd1e374b8760051077a5266b2db',
  'Content-Type': 'application/json'
} 
RequestBody:
 undefined 

handleRestResponse:
 gate GET https://api.gateio.ws/api/v4/options/orders/2597550537?contract=BTC_USDT-20230601-27500-C 404 Not Found 
ResponseHeaders:
 {
  Connection: 'keep-alive',
  'Content-Encoding': 'gzip',
  'Content-Type': 'application/json',
  Date: 'Wed, 31 May 2023 10:37:19 GMT',
  Server: 'openresty',
  'Transfer-Encoding': 'chunked'
} 
ResponseBody:
 {"label":"ORDER_NOT_FOUND","message":"order not found","code":210}
 
```